src/components/IdCard.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+function IdCard(props) {
+  const { lastName, firstName, gender, height, birth, picture } = props
+    return (
+    <div>
+        <h3> {firstName}</h3>
+        <h3> {lastName}</h3>
+        <p>Gender: {gender}</p>
+        <p>Height: {height}</p>
+        <p>Birth: {birth.toLocaleString().slice(0,9)}</p>
```

Comment:
You missed a number here, it should be:

`<p>Birth: {birth.toLocaleString().slice(0,10)}</p>`